apps/elasticsearch/templates/services/app/configs/set-index-template.sh.j2

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+MAX_RETRY=20
+RETRY_DELAY=5
+c=0
```

Comment:
From my last remark:

```suggestion
declare -i c=0
```
(i think shellcheck will complain about declarating and assigning separately)

Reply:
The same applies for all variables declared above.